NimbleMedium/Sources/Domain/UseCases/Auth/LogoutUseCase.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  LogoutUseCase.swift
+//  NimbleMedium
+//
+//  Created by Minh Pham on 11/10/2021.
+//
+
+import RxSwift
+
+// sourcery: AutoMockable
+protocol LogoutUseCaseProtocol: AnyObject {
+
+    func execute() -> Completable
+}
+
+final class LogoutUseCase: LogoutUseCaseProtocol {
+
+    private let userSessionRepository: UserSessionRepositoryProtocol
+
+    init(
+        userSessionRepository: UserSessionRepositoryProtocol
+    ) {
```

Comment:
We normally add a new line for each parameter in case the line length is more than 120 or there are more than 3 parameters 😁

```suggestion
    init(userSessionRepository: UserSessionRepositoryProtocol) {
```